content/protocols/index.textile

```diff
@@ -0,0 +1,57 @@
+---
+title: Protocols
+meta_description: "Clients can use the Ably network protocol adapters. This is especially useful where an Ably Client Library SDK is not available for your language of choice, or where platform resource constraints prohibit use of a Client Library SDK."
+---
+
+Ably Client Library SDKs are the recommended method for connecting to Ably because they offer support for a comprehensive set of Ably features, such as automatic "connection management":/connect, "authentication token renewal":/auth/token and "presence":/presence-occupancy.
+
+Protocol adapters offer an alternative method for connecting to Ably. The advantage to protocol adapters is that they require fewer resources in terms of memory and network overhead such as in smaller footprint devices, or on a platform where an Ably Client Library SDK isn't available such as an Arduino-based IoT wearable. The potential drawback to consider when evaluating protocol adapters is that they do not support the full set of Ably features, for example the MQTT protocol adapter does not support presence, and the SSE protocol adapter does not support automatic token renewal.
+
+A full list of Ably Client Library SDKs can be found on the "SDK page":https://ably.com/download, and features supported by each SDK can be found in the "features matrix":https://ably.com/download/sdk-feature-support-matrix.
```

Comment:
Left this for now, can properly talk about it when you're back.

Reply:
Let's keep it in for now in that case, but use placeholder links since these will be updated to be docs links shortly. That way it'll remind us to update this sentence before pushing this live.